src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends AbstractFigure {
+    private double radius;
+
+    public Circle(Colors color) {
+        super(color);
+        Random random = new Random();
+        radius = 10 * random.nextDouble();
```

Comment:
It would be better to add `radius`  to constructor arguments and add `this.radius = radius` here

Reply:
The radius value is better to be determined during the creation of the figure. 

The same applies to all other figures.